tools/uart_hw_test/README.md

```diff
@@ -40,8 +40,21 @@ It is MANDATORY to first run the uart_hw_test tash command and enter the port nu
 After this run the ./uart_test.sh test app on linux and enter the port number.
 If this sequence is not followed, then the result is not proper.
 
+### How to USE for windows
+1. Find the UART port to be tested. For example, on the RTK8721 board, we will perform uart test for the uart1.
+2. Connect the uart (Rx, Tx, Gnd) lines from the test device to the Windows PC using Serial-USB converter.
+3. Enable CONFIG_UART_HW_TEST config. Build and download the binary.
+4. After boot up, run the uart_hw_test command at the tash prompt (find the appropiate console for windows to get tash prompt). In our example, we use putty console and enter appropriate serial COM number and open the tash prompt .
+5. Input the appropriate port number . In our example, the uart1 appears as /dev/ttys2 on the device . On the PC, the Serial-USB converter appears as /dev/ttyUSB2 . Hence the port number is 2 in both cases.
```

Comment:
Please update this comment. The Serial-USB converter is a COM port as mentione din point 7